### PR TITLE
update tools site

### PR DESCRIPTION
**Pull request by rettinghaus**
This updates the Tools page and adds a manually compiled table of contents.

### CODE REVIEW

File: resources/tools.md [hunk 1]
```diff
@@ -22,50 +23,64 @@ The MEI schema can be customized for various needs. A customization can be used
 
 On the other hand, a customization can also be useful for extending the schema in order to include features not already available in MEI. This can be desirable when doing developments or experiments with the schema but in can also be used in "production" for projects that have very specific needs for features that are not expected to be included in "out-of-the-box" MEI.
 
-We provide an [online customization](http://custom.simssa.ca) service that makes it easy to generate a schema (or other output formats) from an MEI source file and a customization file. The source code for the customization service itself is available on [GitHub](https://github.com/music-encoding/customeization) 
+We provide an [online customization](http://custom.simssa.ca) service that makes it easy to generate a schema (or other output formats) from an MEI source file and a customization file. The source code for the customization service itself is available on [GitHub](https://github.com/music-encoding/customeization)
```
Comment (ahankinson) [inline resources/tools.md]: This should be `custom.music-encoding.org`.

File: resources/tools.md [hunk 2]
```diff
@@ -22,50 +23,64 @@ The MEI schema can be customized for various needs. A customization can be used
...
+### MEISE
 
-Currently, a completely new version of MEISE, coordinated by the Musicology Seminar Detmold/Paderborn within the project scope of DARIAH-DE is developed in cooperation with several editor and rendering approaches throughout the MEI community. Contact: Nikolaos Beer, University of Paderborn, Musicology Seminar Detmold/Paderborn, [nikolaos.beer@upb.de](mailto:nikolaos.beer@upb.de)
\ No newline at end of file
+The _MEI Score Editor_ (MEISE) is an eclipse-based music notation editor software for viewing and editing encoded music in CWN (Common Western Notation) within MEI files. It facilitates e.g. proof reading or editing and visualization of variants and readings. MEISE is currently available in two versions. Both have been developed at the [Musicology Seminar Detmold/Paderborn](http://muwi-detmold-paderborn.de/ "Muiscology Seminar Detmold/Paderborn") as part of the German eHumanities projects [TextGrid](https://www.textgrid.de/ "TextGrid") and [DARIAH-DE](https://de.dariah.eu/ "DARIAH-DE").
```
Comment (ahankinson) [inline resources/tools.md]: This should probably be updated to point to the online version of Meise -- http://meise.de.dariah.eu

Comment (ahankinson) [reply resources/tools.md]: And the old version should be removed, since it's not very well supported.

File: resources/tools.md [hunk 3]
```diff
@@ -22,50 +23,64 @@ The MEI schema can be customized for various needs. A customization can be used
...
+
+MEISE 1.1 can be downloaded at [sourceforge.net](https://sourceforge.net/projects/meise/ "MEISE at sourceforge.net"). Important introductory information on system requirements and installation as well as a first steps guide are provided at the [wiki section](https://sourceforge.net/p/meise/wiki/Home/ "MEISE Wiki at sourceforge.net"). Furthermore, use case and workflow descriptions can be found within [MEISE's project page](https://de.dariah.eu/mei-score-editor) at the DARIAH-DE website.  
+
+Currently, a completely new version of MEISE, coordinated by the Musicology Seminar Detmold/Paderborn within the project scope of DARIAH-DE is developed in cooperation with several editor and rendering approaches throughout the MEI community. Contact: Nikolaos Beer, University of Paderborn, Musicology Seminar Detmold/Paderborn, [nikolaos.beer@upb.de](mailto:nikolaos.beer@upb.de)
+
+### meiView
```
Comment (ahankinson) [inline resources/tools.md]: We should probably remove this, as it hasn't been updated in four years so it's effectively abandoned.

Comment (rettinghaus) [reply resources/tools.md]: I commented it out for now.

**Pull request merged**